client/extensions/woocommerce/app/settings/shipping/shipping-origin.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import Notice from 'components/notice';
+import ShippingHeader from './shipping-header';
+
+class ShippingOrigin extends Component {
+	constructor( props ) {
+		super( props );
+
+		//TODO: use redux state and real data
+		this.state = {
+			address: {
+				name: 'Octopus Outlet Emporium',
+				street: '27 Main Street',
+				city: 'Ellington, CT 06029',
+				country: 'United States'
+			},
+		};
+	}
+
+	render() {
+		const { translate } = this.props;
+
+		return (
+			<div>
+				<ShippingHeader
+					label={ translate( 'Shipping Origin' ) }
+					description={ translate( 'The address of where you will be shipping from.' ) } />
+				<Notice
```

Comment:
Do we need to show this notice here? Doesn't seem to add too much extra context - the `The address of where you will be shipping from.` description seems like enough.